src/graphql/root/mutation/intraledger-payment-send.ts

```diff
@@ -23,7 +23,11 @@ const IntraLedgerPaymentSendMutation = GT.Field({
   args: {
     input: { type: GT.NonNull(IntraLedgerPaymentSendInput) },
   },
-  resolve: async (_, args, { user, logger }) => {
+  resolve: async (
+    _: void,
```

Comment:
I guess void is not intended for this with `_` should be enough

Reply:
typescript shows a warning, so the `: void` intend is to remove the warning.

![image](https://user-images.githubusercontent.com/11156370/147969818-2af3517a-8c07-45e2-806c-6f27a4636631.png)